docs/build_and_test.md

```diff
@@ -1,7 +1,9 @@
 ## 1.Clone RDASApp
+If running on Orion, you will also need to run `module load git-lfs` before cloning. 
 ```
 git clone --recurse-submodules https://github.com/NOAA-EMC/RDASApp.git
 ```
+
```

Comment:
This extra empty line may not be needed as the code block and heading (##) already leaves lots of space between lines.